documentation/asciidoc/computers/camera/rpicam_apps_building.adoc

```diff
@@ -178,6 +178,23 @@ Finally, run the following command to install your freshly-built `rpicam-apps` b
 $ sudo meson install -C build
 ----
 
+[NOTE]
+======
+rpicam-apps is pre-installed on current 64-bit OS distributions. Before the installation steps above you should remove the pre-installed version.
```

Comment:
I believe it's pre-installed on all current distributions, not just 64-bit?

Reply:
It could be that it is installed on all current installations, I cannot very it. It it is the case, than:

rpicam-apps is pre-installed on current OS distributions. Before the installation steps above you should remove the pre-installed version.